web_src/less/_package.less

```
@@ -0,0 +1,3 @@
+.container-labels td:nth-child(2) {
+  word-break: break-word;
```

```suggestion
  overflow-wrap: anywhere
```

`word-break: break-word` is deprecated [as per MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break)

---

`overflow-wrap: anywhere;` works, thanks!

---

Ah yes, it needs `anywhere` because the string don't contain whitespace.